exercises/armstrong-numbers/description.md

```diff
@@ -0,0 +1,10 @@
+An [Armstrong number](https://en.wikipedia.org/wiki/Narcissistic_number) is a number that is the sum of its own digits each raised to the power of the number of digits.
+It also depends on the base of the number being used (for example base = 10 for the decimal system or base = 2 for the binary system).
+
+For example:<sup></sup>
+153<sub>10</sub> is &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;an armstrong number in base 10, because: 153<sub>10</sub> = 1<sup>3</sup> + 5<sup>3</sup> + 3<sup>3</sup> = 1 + 125 + 27 = 154<sub>10</sub>
```

Comment:
Why six non-breaking spaces in a row?

Reply:
It also doesn't render [well](https://github.com/pheanex/problem-specifications/blob/091476825e82ed4a7f5255f8c96d3464e56ec71d/exercises/armstrong-numbers/description.md).

Reply:
The idea was to keep the indentation for the two example the same, so that differences are more obvious. => Removed